server/src/com/cloud/ha/HighAvailabilityManagerImpl.java

```diff
@@ -264,6 +265,11 @@ public void scheduleRestartForVmsOnHost(final HostVO host, boolean investigate)
                 "Host [" + hostDesc + "] is down." + ((sb != null) ? sb.toString() : ""));
 
         for (VMInstanceVO vm : reorderedVMList) {
+            ServiceOfferingVO vmOffering = _serviceOfferingDao.findById(vm.getServiceOfferingId());
```

Comment:
@alexandrelimassantana I don't see where ``vmOffering`` is being used only for debug.  It is being used on line 269 to determine if the VM is using local storage -- before debugging logging code.